src/api.js

```diff
@@ -10,6 +10,12 @@ export default class API extends mapboxgl.Control {
 
   constructor() {
     super();
+    this.types = {
```

Comment:
Should these be on this or should we have a constants object on the Draw constructor?